src/Router.jsx

```diff
@@ -10,6 +11,7 @@ const Json = React.lazy(() => import('./components/json-schemas/JsonSchemas'));
 const BugBounty = React.lazy(() => import('./components/bounty/Bounty/Bounty'));
 const Quickstart = React.lazy(() => import('./components/quickstart/Quickstart/Quickstart'));
 
+
```

Comment:
extra new line, not needed.
Did you setup Prettier correctly? it should remove this for you automatically.